File: Python/excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        res = ''
        
        while n:
            res = chr((n - 1) % 26 + ord('A')) + res
            n = (n - 1) // 26

        return res

File: Swift/excel-sheet-column-title.swift
class Solution {
    func convertToTitle(_ n: Int) -> String {
        var n = n
        var res = ""
        
        while n > 0 {
            let unicodeStr = UnicodeScalar(UInt32((n - 1) % 26) + UnicodeScalar("A")!.value)!
            res = String(unicodeStr) + res
            n = (n - 1) / 26
        }
        
        return res
    }
}
